compiler/noirc_evaluator/src/ssa/ssa_gen/context.rs

```diff
@@ -1000,6 +1000,8 @@ fn convert_operator(op: BinaryOpKind) -> BinaryOp {
         BinaryOpKind::Xor => BinaryOp::Xor,
         BinaryOpKind::ShiftLeft => BinaryOp::Shl,
         BinaryOpKind::ShiftRight => BinaryOp::Shr,
+        //TODO(totel) Make sure all Implication operations are optimized away.
+        BinaryOpKind::Implication => unreachable!(),
```

Comment:
The comment is not entirely accurate. Instead of a comment, this message should be a string inside `unreachable`.